### PR TITLE
feat: fetch original data

### PR DESCRIPTION
**Pull request by sebbalex**
This pr aims to retrieve the original opendata published [here](https://github.com/italia/covid19-opendata-vaccini)

### CODE REVIEW

File: src/loadData.js [hunk 1]
```diff
@@ -87,7 +84,7 @@ const elaborate = (data) => {
     gen_f: categoriesAndAges.reduce(sumDoseX("sesso_femminile"), 0),
   };
 
-  const timestamp = categoriesAndAges[0].data;
+  const timestamp = categoriesAndAges[0].ultimo_aggiornamento;
```
Comment (bfabio) [inline src/loadData.js:87]: We can use data from `https://raw.githubusercontent.com/italia/covid19-opendata-vaccini/master/dati/last-update-dataset.json` here

Comment (sebbalex) [reply src/loadData.js:87]: wow sure

File: src/loadData.js [hunk 2]
```diff
@@ -87,7 +85,7 @@ const elaborate = (data) => {
...
```
Comment (bfabio) [inline src/loadData.js:88]:
```suggestion
  const timestamp = data.dataLastUpdate.ultimo_aggiornamento;
```

File: src/loadData.js [hunk 3]
```diff
@@ -129,5 +129,6 @@ export const loadData = async () => {
       dataPointsSommSummary,
```
Comment (bfabio) [inline src/loadData.js:129]:
```suggestion
      dataPointsSommSummary,
      dataLastUpdate,
```

File: src/loadData.js [hunk 4]
```diff
@@ -129,5 +129,6 @@ export const loadData = async () => {
...
     }),
     dataVaxLocations,
+    dataLastUpdate,
```
Comment (bfabio) [inline src/loadData.js:132]:
```suggestion
```

**Review by bfabio — CHANGES_REQUESTED** [review]
We need to use the field from the new file

**Comment by sebbalex** [comment]
thanks @bfabio 😱

**Review by bfabio — APPROVED** [review]

**Pull request merged**